Formula/unbound.rb

```diff
@@ -13,11 +13,13 @@ class Unbound < Formula
 
   depends_on "openssl"
   depends_on "libevent"
+  depends_on "expat"
```

Comment:
Do you have a `gist-logs` for that build at all, by any chance? It's entirely possible the OS X `expat` is too old now, just nice to check we're not adding dependencies unnecessarily.